content/en/docs/concepts/configuration/secret.md

```diff
@@ -68,7 +68,7 @@ help automate node registration.
 ### Use case: dotfiles in a secret volume
```

Comment:
Aside: this conflates “secret” with “hidden file”, and for that reason it's a poor choice for our first example.

This PR shouldn't be where we fix that, though.